.travis.yml

```diff
@@ -2,10 +2,16 @@ language: php
 
 sudo: false
 
-php: [5.3, 5.4, 5.5, 5.6, nightly, hhvm]
```

Comment:
don't drop CI entirely for older versions. We must keep CI for all versions supported in the composer.json. Use an explicit `dist` for jobs on older versions to run them on compatible environments (`trusty` for 5.4 and 5.5, and `precise` for 5.3).

Also, I would keep the list here for all jobs not needing a specific config in the matrix. It makes it more compact.

Reply:
@snapshotpl can you restore this please?

Reply:
Sure, however phpunit contains compatibility break between php 5.3 and 7.3 versions. For example namespaces

Reply:
we can use namespaces for TestCase in all versions we need, as they are available in 4.8.35+